.travis.yml

```diff
@@ -2,6 +2,10 @@ dist: trusty
 sudo: false
 language: c
 
+env:
+  global:
+    - secure: "YJSSinoWNfMicNYCdxsnlgZc9AIi1yKkgJTmKk41zRI23r8c5Z339Zl1p/boxOxD+YLCQshBYR00ckJXT9R8wQy0WhpWdA3YJYAMWjnVkbBMBNf3VEQSIvHwV8kLDBMPBikhks7b4ACPG3UYLsQrxUJFfJz4Grn7z8tbUedv1Gk="
```

Comment:
It's not secure now. Does this need to be there?

Reply:
Should be secure, can you read it ?

Reply:
Just being funny.